src/App.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.icons-con {
+  display: flex;
+  align-items: center;
+}
+
+.icons-con i {
+  color: grey;
+  font-size: 24px;
+  margin-right: 12px;
+  display: inline-block;
+}
+
+.columns {
+  display: flex;
+  justify-content: center;
+}
+
+.margin-top {
+  margin-top: 2rem;
```

Comment:
- [ ] Hi, You have done a great job. Now I would advise you to follow the design requested by the project:
- Can you follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
   * Typography: font face, size, and weight
   * Layout: composition and space between elements.
You can select one main color for the app and be consistent
- Can you provide the "Back" button (<), When the user clicks on the user navigates to the home page?

- [ ] See your Application:

- Home page
![Capture1](https://user-images.githubusercontent.com/63972670/227641024-eb712ce9-d5d1-4334-8e17-88fa3edb88d4.PNG)

- Detail page

![1](https://user-images.githubusercontent.com/63972670/227641242-b9d031fe-4b1d-44fd-87d5-aa977e053b46.PNG)

- [ ] Original Design:

![Capture](https://user-images.githubusercontent.com/63972670/227641607-47a18aa2-7fe0-47ce-a4a3-83dbed612a2d.PNG)